templates/prometheus-prometheusrules.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if and (.Values.telemetry.prometheusOperator.enabled) (.Values.telemetry.prometheusOperator.prometheusRules.enabled) (.Values.telemetry.prometheusOperator.prometheusRules.rules) }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "vault.fullname" . }}
+  labels:
+    helm.sh/chart: {{ include "vault.chart" . }}
+    app.kubernetes.io/name: {{ include "vault.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    {{- $selectors := .Values.telemetry.prometheusOperator.prometheusRules.selectors }}
+    {{- if $selectors }}
+    {{- toYaml $selectors | nindent 4 }}
+    {{- else }}
+    {{/*
+    update the selectors docs in values.yaml whenever these defaults change.
+    */}}
+    app: kube-prometheus-stack
```

Comment:
It looks like this default label was removed from the ServiceMonitor, so I wonder if it needs to be here too? It seems like the app is defined by the `app.kubernetes.io/name` label above, and from what I can tell the default prometheus install only selects on `release: prometheus`.

```suggestion
```

Reply:
Fixed in 4ee36553b854d76b6b2f69980d9d0e643e2dd9e8

Reply:
Ah, yes this was a part of the original PR. I wonder if it might make sense to make the `release` value public. We we can set the default in the template here.

Reply:
> I wonder if it might make sense to make the `release` value public

Leaving it with the default like you have it seems fine.